src/components/usertask-component/dynamic-usertask-component.tsx

```diff
@@ -43,53 +46,54 @@ export class DynamicUserTaskComponent {
 
   @Watch('usertask')
   public watchUserTaskHandler(newUserTask: IUserTask, oldUserTask: IUserTask): void {
-    this._formFields = [];
+    this.formFields = [];
 
     const hasUserTask: boolean = newUserTask !== undefined && newUserTask !== null;
 
     if (hasUserTask) {
       for (const formField of newUserTask.data.formFields) {
-        const component: any = this._createComponentForFormField(formField);
+        const component: any = this.createComponentForFormField(formField);
         component.componentWillLoad();
-        this._formFields.push(component);
+        this.formFields.push(component);
       }
     }
   }
 
-  public render(): any {
+  public render(): JSX.Element {
     const hasUserTask: boolean = this.usertask !== undefined && this.usertask !== null;
     if (hasUserTask) {
-      const isConfirmUserTask: boolean = this.usertask.data.preferredControl === 'confirm';
+      // eslint-disable-next-line dot-notation
+      const isConfirmUserTask: boolean = this.usertask.data['customForm'] === 'confirm' || this.usertask.data.preferredControl === 'confirm';
```

Comment:
Ich würde hier `??` verwenden, um sicherzugehen, dass `preferredControl` wirklich nur evaluiert wird, wenn in `customForm` nichts drin steht.

```suggestion
      const isConfirmUserTask: boolean = this.usertask.data['customForm'] === 'confirm' ?? this.usertask.data.preferredControl === 'confirm';
```

Reply:
Wenn customForm hier null oder undefined wäre würde aus `isConfirmUserTask ` false werden, auch wenn preferredControl auf confirm steht